dklearn/wrappers.py

```diff
@@ -0,0 +1,256 @@
+from __future__ import division, print_function, absolute_import
+
+from functools import partial
+
+import numpy as np
+import dask
+import dask.array as da
+import dask.bag as db
+from dask import threaded
+from dask.base import Base
+from dask.delayed import delayed, Delayed
+from sklearn.base import clone, is_classifier
+from sklearn.exceptions import NotFittedError
+from sklearn.utils.metaestimators import if_delegate_has_method
+
+from .core import (LazyDaskEstimator, check_X_y, as_lists_of_delayed,
+                   is_list_of)
+from .utils import copy_to
+
+
+__all__ = ('DelayedEstimator', 'Averaged', 'Chained')
+
+
+def _finalize_estimator(res, base):
+    base = clone(base)
+    if hasattr(base, '_update_estimator'):
+        return base._update_estimator(res[0])
+    return copy_to(res[0], base)
+
+
+class DelayedEstimator(Base):
```

Comment:
I'm still somewhat in favor of removing this class if we can do so cleanly, instead going with an xarray style of holding on to lazy results.  This preference isn't based on particularly deep experience though.  Thoughts?